lib/foreman_rh_cloud/engine.rb

```diff
@@ -121,8 +121,12 @@ def self.register_scheduled_task(task_class, cronline)
     end
   end
 
-  def self.with_local_advisor_engine?
-    SETTINGS.dig(:foreman_rh_cloud, :use_local_advisor_engine) || false
+  def self.with_insights_smartproxy?
```

Comment:
Smart Proxy is two words.

```suggestion
  def self.with_insights_smart_proxy?
```